framework/src/meshgenerators/FancyExtruderGenerator.C

```diff
@@ -123,6 +136,35 @@ FancyExtruderGenerator::FancyExtruderGenerator(const InputParameters & parameter
     for (unsigned int j = 0; j < elevation_swaps.size(); j += 2)
       elevation_swap_pairs[elevation_swaps[j]] = elevation_swaps[j + 1];
   }
+
+  if (_elem_integers_swaps.size() &&
+      (_elem_integers_swaps.size() != num_elevations * _elem_integer_names_to_swap.size()))
+    paramError("elem_integers_swaps",
+               "If specified, 'elem_integers_swaps' must be the same length as the product of the "
+               "length of 'heights' and the length of 'elem_integer_names_to_swap' in ",
+               name());
```

Comment:
```suggestion
               "length of 'heights' and the length of 'elem_integer_names_to_swap'");
```

`paramError()` provides context (either input line name, or object name and type if created via an action) so the additional context here is extraneous.